src/test/java/org/mongojack/TestJavaTimeHandling.java

```diff
@@ -0,0 +1,701 @@
+                org.bson.types.ObjectId.class);
+
+        // enable as timestamps.
+        coll.disable(JacksonDBCollection.Feature.WRITE_DATES_AS_TIMESTAMPS);
+
+        // save the object
+        coll.insert(object);
+
+        // retrieve it
+        OffsetDateTimeContainer result = coll.findOneById(id);
+
+        // verify it
+        assertThat(result._id, equalTo(id));
+        assertThat(result.offsetDateTime, equalTo(object.offsetDateTime));
+    }
+
+    public static class DurationContainer {
+        public org.bson.types.ObjectId _id;
+        public Duration duration;
+    }
+
+    /*
+        Like ZonedDateTime, Duration serializes to a BigDecimal when writing as a timestamp, since MongoJack does
+        not serialize BigDecimals, Duration serialization as a timestamp is not supported.
+
+    @Test
+    public void testDurationSavedAsTimestamps() {
+        // create the object
+        DurationContainer object = new DurationContainer();
+        org.bson.types.ObjectId id = new org.bson.types.ObjectId();
+        object._id = id;
+        object.duration = Duration.ofMinutes(2047);
+
+        // get a container
+        JacksonDBCollection<DurationContainer, org.bson.types.ObjectId> coll = getCollection(DurationContainer.class,
+                org.bson.types.ObjectId.class);
+
+        // enable as timestamps.
+        coll.enable(JacksonDBCollection.Feature.WRITE_DATES_AS_TIMESTAMPS);
+
+        // save the object
+        coll.insert(object);
+
+        // retrieve it
+        DurationContainer result = coll.findOneById(id);
+
+        // verify it
+        assertThat(result._id, equalTo(id));
+        assertThat(result.duration, equalTo(object.duration));
+    }
+    */
+
+    @Test
+    public void testDurationSavedAsISO8601() {
+        // create the object
+        DurationContainer object = new DurationContainer();
+        org.bson.types.ObjectId id = new org.bson.types.ObjectId();
+        object._id = id;
+        object.duration = Duration.ofMinutes(2047);
+
+        // get a container
+        JacksonDBCollection<DurationContainer, org.bson.types.ObjectId> coll = getCollection(DurationContainer.class,
+                org.bson.types.ObjectId.class);
+
+        // enable as timestamps.
+        coll.disable(JacksonDBCollection.Feature.WRITE_DATES_AS_TIMESTAMPS);
+
+        // save the object
+        coll.insert(object);
+
+        // retrieve it
+        DurationContainer result = coll.findOneById(id);
+
+        // verify it
+        assertThat(result._id, equalTo(id));
+        assertThat(result.duration, equalTo(object.duration));
+    }
+
+    public static class InstantContainer {
+        public org.bson.types.ObjectId _id;
+        public Instant instant;
+    }
+
+    /*
+        Like ZonedDateTime, an Instant serializes to a BigDecimal when writing dates as timestamps.  Since BigDecimals
+        are not supported by MongoJack, neither can we support serialization of Instants.
+
+    @Test
```

Comment:
you could tag this as `@Ignore` instead of commenting it out

Reply:
same for the tests above

Reply:
I can do that if you'd prefer, but I would worry that a developer might mistake the presence of a test as an implication that it is somehow supported?